cmd/kubeadm/app/cmd/init.go

```diff
@@ -72,7 +71,7 @@ var (
 		You can now join any number of machines by running the following on each node
 		as root:
 
-		  kubeadm join --token {{.Token}} {{.MasterIP}}:{{.MasterPort}} --discovery-token-ca-cert-hash {{.CAPubKeyPin}}
+		  kubeadm join --token {{.Token}} {{.MasterEndpoint}} --discovery-token-ca-cert-hash {{.CAPubKeyPin}}
```

Comment:
@ivan4th thanks for catching this. Let me look into the best way to update the PR. I'll keep you posted.